src/Phinx/Db/Adapter/SqlServerAdapter.php

```diff
@@ -482,7 +482,11 @@ public function getColumns($tableName)
      */
     protected function parseDefault($default)
     {
-        $result = preg_replace(["/\('(.*)'\)/", "/\(\((.*)\)\)/", "/\((.*)\)/"], '$1', $default);
+        if ($default === null) {
```

Comment:
Then docblock should state also null as possible param value

Reply:
Done in 593af76